lte/gateway/python/magma/pipelined/qos/common.py

```diff
@@ -180,24 +180,21 @@ def init_impl(self, datapath):
         """
         Takese in datapath, and initializes appropriate QoS manager based on config
         """
+        if self._initialized:
```

Comment:
can you also move lock to object attribute from class attribute, you could create separate PR to address this.